### ISSUE TITLE
Simplify CocoaPod integration

### ISSUE THREAD

**Issue by krzysztofzablocki** [krzysztofzablocki/Sourcery#556]
I think we can use newly released script phases to get rid of manual step in Sourcery integration http://blog.cocoapods.org/CocoaPods-1.4.0/

**Comment by ilyapuchka** [comment]
What script would we put there then? Just `$PODS_ROOT/Sourcery/bin/sourcery` and rely on config file being in source root?

**Comment by krzysztofzablocki** [comment]
yeah I think so, yml is the way to go anyway, the command line args are error prone and annoying to repeat when people want to work on new templates etc
